test/ga-track.test.js

```diff
@@ -226,6 +226,17 @@ test('Can send arbitrary events to GA', assert => {
   assert.end();
 });
 
+test('Can send arbitrary data to GA', assert => {
+  setup();
+
+  GATrack.set('some', 'random', 'stuff');
```

Comment:
Set is sort of an odd method name for this

Reply:
Any other suggestions? AFAIK this is used to set config so it seemed appropriate.

Reply:
I don't see this as setting config because you can call it multiple times. Config is usually a set once and persist